src/webgpu/api/validation/capability_checks/features/timestamp_query.spec.ts

```diff
@@ -0,0 +1,40 @@
+export const description = '';
+
+import { params, pbool, poptions } from '../../../../../common/framework/params_builder.js';
+import { makeTestGroup } from '../../../../../common/framework/test_group.js';
+import { ValidationTest } from '../../validation_test.js';
+
+export const g = makeTestGroup(ValidationTest);
+
+g.test('create_query_set')
+  .desc(
+    `
+Tests that create query set with the type of timestamp shouldn't be valid without timestamp query
+enabled.
+- create query set with the type { occlusion, pipeline-statistics, timestamp }
+- x = timestamp query { enable, disable }
```

Comment:
Perhaps it would make sense to merge these two files and have a single test since they are so similar. It could even test combinations of timestampQueryEnable and pipelineStatisticsQueryEnable.

Reply:
I saw that it requires one file for each feature name in README.

Reply:
I wrote that README without realizing some tests would combine nicely. We can change that guidance. OTOH this is fine.

Reply:
Also it's likely we'll eventually have to test validation with combinations of extensions at some point, so I think we'll have to relax that someday anyway.